scripts/backend_tests.py

```diff
@@ -39,6 +39,7 @@
 ALL_ERRORS = []
 # This should be the same as core.test_utils.LOG_LINE_PREFIX.
 LOG_LINE_PREFIX = 'LOG_INFO_TEST: '
+_LOAD_TESTS_DIR = 'core/tests/load_tests'
```

Comment:
If we want this to work on non-unix systems, this should probably be os.path.join('core', 'tests', load_tests').